### ISSUE TITLE
DEPRECATION: `Enumerable#contains` is deprecated, use `Enumerable#includes` instead

### ISSUE THREAD

**Issue by rizkytahara** [kaliber5/ember-bootstrap#132]
Hi,

I got:

```
DEPRECATION: `Enumerable#contains` is deprecated, use `Enumerable#includes` instead. [deprecation id: ember-runtime.enumerable-contains] See http://emberjs.com/deprecations/v2.x#toc_enumerable-contains for more details.
```

seems the `array-includes.js` didn't recognize the existance of the function, so it goes to use `arrayContains` again...

```
$ ember -v
ember-cli: 2.8.0
node: 4.4.7
os: linux x64
```

I'm using: "ember-bootstrap": "0.9.1"

any fix for this?

**Comment by simonihmig** [comment]
Yes, have already seen this as well. Fix is in the works...

**Comment by simonihmig** [comment]
Should be fixed in v0.10.0, just released!
